pubspec.yaml

```diff
@@ -142,6 +144,7 @@ flutter:
     - packages/smartcontracts/contracts/apps/SimpleFaucet.abi.json
     - packages/smartcontracts/contracts/standards/ERC20.abi.json
     - packages/smartcontracts/contracts/apps/Profile.abi.json
+    - packages/smartcontracts/contracts/external/CardFactory.abi.json
```

Comment:
What about the CardManagerModule abi?

You could import it similar to how we import the other smart contracts. 

But it's a new repo:
https://github.com/citizenwallet/contract-forge/blob/main/abi/CardManagerModule.json
```
https://github.com/citizenwallet/contract-forge.git
      ref: v0.0.1
```

`packages/contract-forge/abi/CardManagerModule.json`